_episodes/02-running-containers.md

```diff
@@ -27,9 +27,43 @@ singularity --help
 
 # Downloading Images
 
-The [Singularity Container Library](https://cloud.sylabs.io/library) is the official image registry.
-Images built by users are accessible using the CLI, and images become containers at runtime.
+Apptainer/Singularity can store, search and retrieve images in registries.
+Images built by other users can be accessible using the CLI, can be pulled down, and become containers at runtime.
 
+Sylabs, the developers of one Singularity flavor, hosts a public image registry, the
+[Singularity Container Library](https://cloud.sylabs.io/library) where many user built images are available.
+
+The Linux Foundation singularity, apptainer does not point by default to the Sylab registry as previous versions did.
```

Comment:
Thanks for the additions! I feel this first sentence is not clear in the sense of calling Apptainer "The Linux Foundation singularity".  What about using "The Linux Foundation flavor, Apptainer, does not point..."

Reply:
done